dll/win32/shell32/folders/CControlPanelFolder.cpp

```diff
@@ -660,6 +660,23 @@ HRESULT WINAPI CControlPanelFolder::GetCurFolder(PIDLIST_ABSOLUTE * pidl)
     return S_OK;
 }
 
+HRESULT WINAPI CControlPanelFolder::MessageSFVCB(UINT uMsg, WPARAM wParam, LPARAM lParam)
+{
+    switch (uMsg)
+    {
+        case SFVM_DEFVIEWMODE:
+        {
+#if ROSPOLICY_SHELLFOLDER_DEFLARGEICONS & ( 1 << (PT_CONTROLS_NEWREGITEM >> 4) )
```

Comment:
Is there no 'nice' macro for the `1<<` and `>> 4`?
Like `PT_CONTROLS_TO_ROSPOLICY` or so? (just making it up as I go here)

Rationale:
You can search for a name in documentation, but good luck searching for a shift in documentation, especially if you don't exactly know what to look for.